### PR TITLE
VTCCA-4451: improve page title and spec as per req

### PR DESCRIPTION
**Pull request by zahiriqbal**

### CODE REVIEW

File: app/views/dvr/cannotRaiseChallenge.scala.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 @(model: CanChallengeResponse, address: String, localAuth: String, homePageUrl: String, authorisationId: Long, backLinkUrl: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
 @mainLayout(
-    pageTitle = messages("cannotRaiseChallenge.title"),
+    pageTitle = messages("cannotRaiseChallenge.title.fail"),
```
Comment (PeterPerhac) [inline app/views/dvr/cannotRaiseChallenge.scala.html:26]: I don't think you should add the `.fail` suffix. it is the cannotRaiseChallenge page, and this is its `title`. Why `.fail`?

Comment (zahiriqbal) [reply app/views/dvr/cannotRaiseChallenge.scala.html]: @PeterPerhac Updated

**Comment by darrell-plant** [comment]
LGTDz 👍

**Comment by PeterPerhac** [comment]
👍  looks good to me

**Pull request merged**